CONTRIBUTING.md

```diff
@@ -41,12 +41,16 @@ repositories each with the same name is a branch set, e.g. the master
 branch in each repository forms the master branch set.
 
 Some general rules to follow:
-
+-   We follow [HubFlow](https://datasift.github.io/gitflow/) which means we use
```

Comment:
I did a poor job of explaining this but didn't have much time.  I think we want to make it clear, though, that we make pull requests against develop.  Develop is our working branch.  Master is used for releases and for the default branch for people wanting to get to our stable stuff.